content/en/account_management/billing/allotments.md

```diff
@@ -0,0 +1,113 @@
+
+For example, if a user is committed to 5 APM Pro Hosts, they have a default Ingested Spans allotment of 5 APM Pro hosts * 150 GB Ingested Spans / host = 750 GB for the month. If they use 6 APM Hosts and 800 GB of Ingested Spans, they are billed for the additional host usage but not for the additional spans usage, since their allotment increases to 900 GB.
+
+### On-demand option
+
+The allotment usage of a product can be computed according to the user's on-demand metering option. Users can opt for a **monthly or hourly** on-demand option. A parent product's on-demand option determines the child's on-demand option; if a parent product has a monthly option, then the child can have either a monthly or hourly option. If a parent product has an hourly option, then the child must also have an hourly option (for example, if there is an hourly APM Host subscription, then the Ingested Spans subscription will also be calculated using an hourly option).
+
+If a user's contract has a subscription specified for a product and the product supports both hourly and monthly on-demand options, we default to a monthly option. By default, the on-demand option is set at the subscription level and applies to all products except for the following, which support a single default. 
+
+| Product                                          | Default option |
+|--------------------------------------------------|----------------|
+| Containers (Infrastructure, Profiled, CSPM, CWS) | Hourly         |
+| Incident Management                              | Monthly        |
+| APM Fargate Products                             | Monthly        |
+| Serverless APM                                   | Monthly        |
+| Logs Products                                    | Monthly        |
+| SNMP Traps                                       | Monthly        |
+
+On hourly metering, the monthly allotment is adjusted to an hourly allotment. For summed products such as APM spans and events, the monthly allotment is annualized and then divided by the number of hours in a year to get the hourly allotment. For leap years, monthly allotments are divided by 732 hrs to get the hourly quantity. For non-leap years, they are by 730 hrs. For averaged products such as metrics, the monthly allotment stays the same at either on-demand option since total monthly usage is the average of billable usage across all hours in the month. For a full list of hourly and monthly default allotments by product, see [Allotments Table][2].
+
+On-demand usage is calculated at the hourly level for this on-demand option as well. Therefore, the total monthly on-demand usage is the sum of hourly on-demand usage. Hourly allotments are not carried over to subsequent hours.
```

Comment:
It wasn't clear to me what "this on-demand option" is referring to here, consider revising unless you are confident this will be clear to your audience.